src/app/extensions/seo/store/seo/seo.effects.ts

```diff
@@ -64,7 +64,12 @@ export class SeoEffects {
             // PRODUCT PAGE
             this.productPage$.pipe(map(product => this.baseURL + generateProductUrl(product).substring(1))),
             // CATEGORY / FAMILY PAGE
-            this.categoryPage$.pipe(map(category => this.baseURL + generateCategoryUrl(category).substring(1))),
+            this.categoryPage$.pipe(
+              mergeMap(category =>
```

Comment:
why `mergeMap` instead of `switchMap`?
It seems like outdated category information could safely be thrown away.